third-party/realdds/src/dds-device.cpp

```diff
@@ -19,11 +19,91 @@ std::mutex devices_mutex;
 }  // namespace
 
 
+dds_device::dds_stream::dds_stream( rs2_stream type, std::string group_name )
```

Comment:
Please don't do this.
Different classes in different files.
Also, I'd like to see this class not be inside dds_device: a stream (and stream-server) should be accessible (constructible, etc.) by themselves.

Reply:
std::string should be passed by const &, please -- unless you intend on changing the local value?

Reply:
We're making things difficult for outselves!
We should not use **any** libRS enums/structs in the code here.

Reply:
Created dds_stream::profile. removed libRS enums from dds_device, dds_stream, notification-msg.h and control-msg.h